webpack/assets/javascripts/react_app/common/MountingService.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import StatisticsChartsList from '../components/charts/StatisticsChartsList';
+import ReactDOM from 'react-dom';
+
+export function mount(component, selector, data) {
+
+  const components = {
+    StatisticsChartsList: {
+      type: StatisticsChartsList,
+      markup: <StatisticsChartsList data={data}/>
+    }
+  };
+
+  const reactNode = document.querySelector(selector);
+
+  if (reactNode) {
+    ReactDOM.render(components[component].markup, reactNode);
+  } else {
+    const componentName = components[component].type.name;
+
+    // eslint-disable-next-line no-console
```

Comment:
? If you want to enable console for certain messages ('error'?) feel free to edit the eslintrc to do `{ "allow": ["error"] }`